C/plugins/storage/sqlite/common/readings.cpp

```diff
@@ -495,19 +497,22 @@ int Connection::readingStream(ReadingStream **readings, bool commit)
 
 						if (sqlite3_resut == SQLITE_LOCKED  )
 						{
-							sleep_time_ms = (1 * PREP_CMD_RETRY_BACKOFF);
+							sleep_time_ms = PREP_CMD_RETRY_BASE + (random() &  PREP_CMD_RETRY_BACKOFF);
```

Comment:
Do you mean to do a logical add here? You are doing an AND operation with 0x1388 which only gives you small range of values. Is it perhaps (random() % PREP_CMD_RETRY_BACKOFF) you want to give you a value between 0 and 4999?

Reply:
the intention is to have 
- a fixed amount of time to wait PREP_CMD_RETRY_BASE 
- plus a randomly generated: (random() &  PREP_CMD_RETRY_BACKOFF) - produces a number between 0 to 5000

Reply:
fixed